portfolio/src/main/webapp/script.js

```diff
@@ -12,48 +12,10036 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+const russianWords = 
```

Comment:
This is an impressive list! Can you describe what the intended use is for the words in this list? Could this be shortened to a list of 10-20 of your favorite words or does it need a lot of entires?

Reply:
I just wanted an extensive list in the case that a potential russian speaker were to visit the site,but, your right it can be shortened

Reply:
Forgot to push?